haddock-api/src/Haddock.hs

```diff
@@ -382,42 +337,9 @@ render log' dflags unit_state flags sinceQual qual ifaces installedIfaces extSrc
 
     visibleIfaces    = [ i | i <- ifaces, OptHide `notElem` ifaceOptions i ]
 
-<<<<<<< HEAD
-    -- /All/ interfaces including external package modules, grouped by
-    -- interface file (package).
-    allPackages      :: [PackageInterfaces]
-    allPackages      = [PackageInterfaces
-                         { piPackageInfo = packageInfo
-                         , piVisibility  = Visible
-                         , piInstalledInterfaces  = map toInstalledIface ifaces
-                         }]
-                    ++ map snd packages
-
-    -- /All/ visible interfaces including external package modules, grouped by
-    -- interface file (package).
-    allVisiblePackages :: [PackageInterfaces]
-    allVisiblePackages = [ pinfo { piInstalledInterfaces =
-                                     filter (\i -> OptHide `notElem` instOptions i)
-                                            piInstalledInterfaces
-                                 }
-                         | pinfo@PackageInterfaces
-                             { piVisibility = Visible
-                             , piInstalledInterfaces
-                             } <- allPackages
-                         ]
-
-    -- /All/ installed interfaces.
-    allInstalledIfaces :: [InstalledInterface]
-    allInstalledIfaces = concatMap (piInstalledInterfaces . snd) packages
-||||||| parent of 1b63771d (coot/multiple package (ghc-head) (#1419))
-    -- /All/ visible interfaces including external package modules.
-    allIfaces        = map toInstalledIface ifaces ++ installedIfaces
-    allVisibleIfaces = [ i | i <- allIfaces, OptHide `notElem` instOptions i ]
-=======
     -- /All/ visible interfaces including external package modules.
     allIfaces        = map toInstalledIface ifaces ++ map snd installedIfaces
     allVisibleIfaces = [ i | i <- allIfaces, OptHide `notElem` instOptions i ]
->>>>>>> 1b63771d (coot/multiple package (ghc-head) (#1419))
```

Comment:
Looking at this merge conflict, it seems you picked the wrong one.  It might be a sign that your branch does not contain all the commits that where already merged and by accident you'll remove some of the work that was committed to master.

Reply:
Ah, thanks. It's a shitshow indeed.